README.md

```diff
@@ -99,6 +99,52 @@ jobs:
           maven_settings_path: .maven/settings.xml
 ```
 
+## Mulesoft Shared Workflows
```

Comment:
How about adding this section @hoangmirs 

```
<p> This contains a collection of <a href="#mulesoft-actions">Mulesoft Actions</a> and a pre-defined <a href="#mulesoft-shared-workflows">Mulesoft Shared Workflows</a> that speed up the development process of Mulesoft projects. The pre-defined workflows are a ready-to-use solution which combines the Actions for running task like MUnit tests or deploying Mulesoft projects.</p>
```

Before the `## Mulesoft Actions` and `## Mulesoft Shared Workflows` so that the user won't be confuse about the usage for both types

![image](https://github.com/nimblehq/mulesoft-actions/assets/11751745/b677481d-1eb9-4793-b8d2-01d094dba085)

Reply:
Adjusted in https://github.com/nimblehq/mulesoft-actions/pull/4/commits/459de797861aa7fb7c860891bf4e04091cd135cc